source/features/enable-file-links-in-compare-view.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+
+function handlePRMenuOpening(event: delegate.Event): void {
+	event.delegateTarget.classList.add('rgh-actionable-link'); // Mark this as processed
+
+	// This solution accounts for:
+	// - Branches with slashes in it
+	// - PRs opened from the default branch
+	const dropdown = event.delegateTarget.nextElementSibling!;
+	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
+	const headBranchUrl = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname;
+	const filepath = dropdown.closest<HTMLDivElement>('[data-path]')!.dataset.path;
+	viewFile.pathname = headBranchUrl + '/' + String(filepath);
+}
+
+function handleCompareMenuOpening(event: delegate.Event): void {
+	event.delegateTarget.classList.add('rgh-actionable-link'); // Mark this as processed
+	const branch = select('[title^="compare"]')!.textContent!;
+
+	const dropdown = event.delegateTarget.nextElementSibling!;
+	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
+	viewFile.before(
+		<div className="dropdown-header pl-5 d-block">
```

Comment:
Try matching the existing elements instead. They don't have `pl-5 d-block`. Does `dropdown-header dropdown-item` work?

Reply:
The dropdown-item causes the text to wrap 
![image](https://user-images.githubusercontent.com/16872793/83419670-70055700-a3f3-11ea-9ae7-54a117db21b3.png)

Reply:
True. This should work though
```suggestion
		<div className="dropdown-header pl-5">
```